packages/serverless-orchestration/src/ServerlessSpoke.js

```diff
@@ -15,11 +15,14 @@ const spoke = express();
 spoke.use(express.json()); // Enables json to be parsed by the express process.
 const exec = require("child_process").exec;
 
-const { Logger, delay } = require("@uma/financial-templates-lib");
-let logger;
+const { delay, createNewLogger, waitForLogger } = require("@uma/financial-templates-lib");
+
+let customLogger;
+
+const waitForLoggerDelay = process.env.WAIT_FOR_LOGGER_DELAY || 5;
 
-const waitForLogsS = 5;
 spoke.post("/", async (req, res) => {
+  const logger = customLogger || createNewLogger();
```

Comment:
I implemented the same logger patten seen in the hub in the spoke. this is not strictly needed here but it was done for two reasons:
1) consistency with the hub
2) in the context you run exactly one spoke (i.e you are running this setup on your local machine) having separate loggers makes things more predictable, as is the case in the hub.